GPU/TPCFastTransformation/Spline1D.cxx

```diff
@@ -13,6 +13,10 @@
 ///
 /// \author  Sergey Gorbunov <sergey.gorbunov@cern.ch>
 
+#if !defined(GPUCA_GPUCODE) && !defined(GPUCA_STANDALONE) // code invisible on GPU and in the standalone compilation
```

Comment:
You don't need GPUCA_GPUCODE here, this file is a cxx and anyway not processed by the GPU

For the standalone: If you want to make it a bit simpler, you can just protect the whole file with GPUCA_STANDALONE, or you can just not include it on the CMake level for the standalone build.
In this way, you avoid the repeated `#ifdef`s

Reply:
But, potentially, a *.cxx file can be also added to the GPU compilation?

Reply:
Sure, but only when added explicitly, and since we will not explicitly add these files that are only meant for ROOT IO, it should be safe. I am just trying to avoid countless `#ifdef`s. But it was just a comment, your call.